uint32 new_exp
---
string message
int8 k4aExposureServiceErrorCode